languages/python/tests/parity/policies/test_api.polar

```diff
@@ -6,19 +6,19 @@ actorInRole(actor, role, resource: Widget) if
     role = resource.company.role(actor);
 
 allow(actor, "get", _: Http{path: path}) if
-    new PathMapper{template: "/widget/{id}"}.map(path) = {id: id} and
+    new PathMapper("/widget/{id}").map(path) = {id: id} and
```

Comment:
If it's still possible to construct with instance literals in python, why have all of these been changed?

Reply:
Because we have shared policy code that uses `PathMapper` and `Http`, they can only use the common denominator of boa constructors.